### DISCUSSION TITLE
[Commands]

### DISCUSSION THREAD

**Discussion by JustIanJ** [codeman1o1/TIJK-Bot#184] [Q&A] [answered]
**What do you want to ask?**
Er zijn veel commands die vgsm niet werken, klopt dit?

**Additional context**
![Screenshot_20211214-221239_Discord.png](https://user-images.githubusercontent.com/90295730/146080171-df20fa89-8804-47c8-840d-5b681eb4f02e.png)

**Comment by JustIanJ** [comment]
Ze staan wel onder .help tho.

**Comment by karstenzip** [comment, marked as answer]
To acces the commands for a specific category do `.help <name>`

**Comment by karstenzip** [comment]
It is case sensitive

**Comment by JustIanJ** [comment]
Zie screenshot .help fun deed niets

**Comment by karstenzip** [comment]
> It is case sensitive

**Comment by JustIanJ** [comment]
Can you fix that?

**Comment by karstenzip** [comment]
Unfortunately not. The help command is provided by a Python package that I do not own.

**Comment by JustIanJ** [comment]
Can you disable the package and code it yourself?i know it's alot of work but i'd appreciate it.

**Comment by karstenzip** [comment]
I won't do that but if you can make an issue for the package here: [https://github.com/squigjess/nextcord-pretty-help](https://github.com/squigjess/nextcord-pretty-help)

**Comment by JustIanJ** [comment]
Can you do it? You know the package better

**Comment by karstenzip** [comment]
You have to do that yourself. It is your idea to make it case insensitive.

**Comment by JustIanJ** [comment]
Nvm
